docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +126,383 @@ microservice reports itself as ready.
 |Treats the instance as still starting up; does not check liveness or readiness until the startup probe reports success or times out according to its configuration.
 |===
 
-=== Built-in and Custom Health Checks
 
-==== Built-in Health Checks
-Helidon provides built-in, default checks for each endpoint.
-The built-in liveness checks include various environmental values, such as whether the JVM has detected deadlocks
-or whether there is sufficient heap space. The built-in readiness and startup checks always report `UP`.
+== Configuration
+
+=== Custom Health Root Path and Port
+
+You can specify a custom port and root context for the root health endpoint path.
+However, you cannot use different ports, such as http://localhost:8080/myhealth and http://localhost:8081/myhealth/live.
+Likewise, you cannot use different paths, such as http://localhost:8080/health and http://localhost:8080/probe/live.
+
+The example below will change the root path.
+
+[source,yaml]
+.Create a file named `application.yaml` in the `resources` directory with the following contents:
+----
+health:
+  web-context: "myhealth"  // <1>
+----
+<1> The web-context specifies a new root path for the health endpoint.
+
```

Comment:
We should be able to remove this section and rely on the generated config doc instead.

Speaking of that, we will need to change either config annotations in health or in service-common or the tool or perhaps all to support inheritance. The `HealthSupport.Builder` extends `HelidonRestServiceSupport.Builder` so the config doc for health will need to incorporate the doc from the superclass. The JSON file the health build generates does not include the superclass config info so something is not quite right. This should be a separate issue from the doc update itself though, except eventually for changing this to include the generated config doc.